packages/auth/amplify_auth_cognito/pubspec.yaml

```diff
@@ -1,18 +1,20 @@
 name: amplify_auth_cognito
 description: The Amplify Flutter Auth category plugin using the AWS Cognito provider.
-version: 0.5.0
-homepage: https://github.com/aws-amplify/amplify-flutter/tree/main/packages/auth/amplify_auth_cognito
+version: 1.0.0-next.0
+homepage: https://docs.amplify.aws/lib/q/platform/flutter/
+repository: https://github.com/aws-amplify/amplify-flutter/tree/next/packages/auth/amplify_auth_cognito
+issue_tracker: https://github.com/aws-amplify/amplify-flutter/issues
 
 environment:
   sdk: ">=2.17.0 <3.0.0"
   flutter: ">=3.0.0"
 
 dependencies:
-  amplify_auth_cognito_android: 0.5.0
-  amplify_auth_cognito_dart: 0.1.0
-  amplify_auth_cognito_ios: 0.5.0
-  amplify_core: 0.5.0
-  amplify_flutter: 0.5.0
+  amplify_auth_cognito_android: 1.0.0-next.0
+  amplify_auth_cognito_dart: ^0.1.0
```

Comment:
nit: we might want to discuss making `amplify_auth_cognito_dart` 1.x-next as well, since it should go 1.0 when everything else does. But that can be addressed later.

Reply:
Yeah, that's probably true - will want Ashish/Abdallah to weigh in as well